### PR TITLE
Added rabbitmq checker

### PR DESCRIPTION
**Pull request by xBlaz3kx**

### CODE REVIEW

**Comment by tavsec** [comment]
@xBlaz3kx looks good, could you just check merge conflicts (I think you can just use your changes)

**Comment by xBlaz3kx** [comment]
Forgot to add docs, I will add those as well

File: README.md [hunk 1]
```diff
@@ -180,6 +180,38 @@ func main() {
 
 ```
 
+### RabbitMQ check
+
+You can perform RabbitMQ connection check using `InfluxV2` checker:
```
Comment (tavsec) [inline README.md:185]: Probably typo `InfluxV2`?

File: README.md [hunk 2]
```diff
@@ -180,6 +180,38 @@ func main() {
...
+func main() {
+	r := gin.Default()
+
+	// Connect to the rabbitmq instance
+	conn, err := amqp.Dial("amqp://guest:guest@localhost:5672/")
+	if err != nil {
+		panic(err)
+	}
+
+	influxCheck := checks.NewRabbitCheck(conn)
```
Comment (tavsec) [inline README.md:207]: Probably typo `influxCheck`?

**Review by tavsec — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (xBlaz3kx) [reply README.md]: Fixed

File: README.md (hunk 1 above)
Comment (xBlaz3kx) [reply README.md]: Fixed

**Pull request merged**